composer.json

```diff
@@ -36,8 +36,8 @@
         "symfony-cmf/resource": "^1.0",
         "symfony-cmf/resource-bundle": "^1.0",
         "symfony-cmf/testing": "^2.0",
-        "symfony/phpunit-bridge": "^2.8 || ^3.0",
-        "symfony/security-acl": "^2.8 || ^3.0"
+        "symfony/phpunit-bridge": "^3.3",
+        "symfony/security-acl": "^2.8 || ^3.2"
```

Comment:
This should be ^3.0

Reply:
Fixed

Reply:
it is still 3.0

Reply:
Looks like a github Problem oO
https://github.com/core23/SonataDoctrinePhpcrAdminBundle/blob/patch/php-support/composer.json#L40